src/error.rs

```diff
@@ -122,18 +122,84 @@ impl fmt::Display for ErrorType {
 #[derive(Debug, PartialEq, Deserialize)]
 #[serde(rename_all = "snake_case")]
 pub enum ErrorCode {
-    InvalidNumber,
-    InvalidExpiryMonth,
-    InvalidExpiryYear,
-    InvalidCvc,
-    InvalidSwipeData,
-    IncorrectNumber,
+    AccountAlreadyExists,
+    AccountCountryInvalidAddress,
+    AccountInvalid,
+    StripeAccount,
+    AccountNumberInvalid,
+    AlipayUpgradeRequired,
+    AmountTooLarge,
+    AmountTooSmall,
+    ApiKeyExpired,
+    BalanceInsufficient,
+    BankAccountExists,
+    BankAccountUnusable,
+    BankAccountUnverified,
+    BitcoinUpgradeRequired,
+    CardDeclined,
+    DeclineCode,
```

Comment:
I don't believe there is a `decline_code` error code;   Decline Codes appear to only be used on a `ChargeOutcome`.